### PR TITLE
Flask Form from Aryan9101

### PR DESCRIPTION
**Pull request by Aryan9101**

### CODE REVIEW

**Review by djmgit — COMMENTED** [review]
Please provide a README.md mentioning how to run the app so that others can learn from your app and can test it.

**Comment by djmgit** [comment]
@Aryan9101 also please provide a suitable commit description. It is always a good practice to provide commit description :)

File: Aryan9101/run.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!flask/bin/python
+from app import app
```
Comment (djmgit) [inline Aryan9101/run.py:2]: try this:

```
if __name__ == "__main__":
  app.run('0.0.0.0', 5005, debug=True)
```

Comment (djmgit) [reply Aryan9101/run.py:2]: @Aryan9101 please try this.

File: Aryan9101/app/views.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+WTF_CSRF_ENABLED = True #Important Config that prevents CSRF attacks
+
+class GreetingForm(Form):
+    """
+    Create a Greeting form class.
+    Contains the main attributes i.e, String fields to be filled in a form
+    """
+    username = StringField('Username', [validators.Length(min = 1)]) #validator to make sure data is entered
+
+@app.route('/')
```
Comment (djmgit) [inline Aryan9101/app/views.py:15]: Try changing this to the following:
```
@app.route('/', methods=['POST', 'GET'])
```

Comment (Aryan9101) [reply Aryan9101/app/views.py:15]: That was there the entire time. I didn't update my PR yet, but in my personal repo, it is there

Comment (djmgit) [reply Aryan9101/app/views.py:15]: So the root route has both GET and POST on the heroku version?

Comment (Aryan9101) [reply Aryan9101/app/views.py:15]: yup

Comment (Aryan9101) [reply Aryan9101/app/views.py:15]: https://github.com/Aryan9101/GCIWebForm

Comment (djmgit) [reply Aryan9101/app/views.py:15]: @Aryan9101 can you please post your heroku logs here

Comment (djmgit) [reply Aryan9101/app/views.py:15]: The new ones after making the previous changes

Comment (djmgit) [reply Aryan9101/app/views.py:15]: The logs when the method not allowed error is thrown

Comment (Aryan9101) [reply Aryan9101/app/views.py:15]: capture2

Comment (djmgit) [reply Aryan9101/app/views.py:15]: Are these the logs after encountering the error?

Comment (djmgit) [reply Aryan9101/app/views.py:15]: @Aryan9101 ^

Comment (Aryan9101) [reply Aryan9101/app/views.py:15]: These are the logs after I changed my run.py. I wasn't getting the OSError anymore, but the application still is not working when deployed.

Comment (djmgit) [reply Aryan9101/app/views.py:15]: @Aryan9101 please try accessing the greeting endpoint or simply refresh your landing page and see whats there in the log

**Pull request merged**